src/sonic-yang-models/yang-models/sonic-bgp-neighbor.yang

```diff
@@ -66,8 +66,11 @@ module sonic-bgp-neighbor {
                         type leafref {
                             path "/lag:sonic-portchannel/lag:PORTCHANNEL/lag:PORTCHANNEL_LIST/lag:name";
                         }
-                        type leafref {
-                            path "/vlan:sonic-vlan/vlan:VLAN/vlan:VLAN_LIST/vlan:name";
+                       // type leafref {
```

Comment:
let's add a comment here; the reason for commenting out VLAN-intf leafref.

Reply:
ok